h2o-py/tests/testdir_misc/pyunit_gh_15614_datatable_2_pandas.py

```diff
@@ -0,0 +1,32 @@
+import sys
+sys.path.insert(1,"../../")
+import h2o
+from tests import pyunit_utils
+from h2o.utils.shared_utils import (can_use_datatable, can_use_pandas)
+import time
+
+# if datable is installed, this test will show that using datatable to convert h2o frame to pandas frame is
+# much faster.  This workaround is suggested by Megan Kurka.  I implemented the code snippet from Bernard Ong.
+def test_datatable_pandas():
+    if can_use_pandas() and can_use_datatable():    # only perform tests if pandas and datatable are installed
```

Comment:
Do we have `datatable` installed in our python testing images?

Reply:
Not sure.  Is this something I can ask the DevOps team to do?  I fixed the typo.